docs/configuration/backends/docker.md

```diff
@@ -292,11 +292,16 @@ If a container is linked to several networks, be sure to set the proper network
 For instance when deploying docker `stack` from compose files, the compose defined networks will be prefixed with the `stack` name.
 Or if your service references external network use it's name instead.
 
-[2] `traefik.frontend.auth.basic.users=EXPR `:  
```

Comment:
could you restore the 2 spaces at the end of line.

Reply:
Oops, sorry. I thought it was whitespaces.